lib/aerosol/launch_configuration.rb

```diff
@@ -90,6 +92,12 @@ def to_s
 }}
   end
 
+  def corrected_user_data
+    realized_user_data = user_data.is_a?(Proc) ? user_data.call : user_data
```

Comment:
If the Proc being called here is using `ERB`, and supplies to `#result` a `binding`, then it has access to the current scope which has access to the meta_data variable of launch_configurations.